.github/workflows/lintfix.yml

```
@@ -0,0 +1,95 @@
+name: lintfix
+
+on:
+  pull_request_target:
+
+env:
+  YARN_MODULES_CACHE_KEY: v1
+  YARN_PACKAGE_CACHE_KEY: v1
+  YARN_CACHE_FOLDER: .cache/yarn
+  NODE_VERSION: 14
+  FIX_AUTHOR_EMAIL: renovate@whitesourcesoftware.com
+
+jobs:
+  lint-fix:
+    runs-on: ubuntu-latest
+
+    # lint shouldn't need more than 10 min
+    timeout-minutes: 15
```

```suggestion
    # lint shouldn't need more than 15 min
    timeout-minutes: 15
```

The comment is not the same as the actual setting.
Maybe we can just remove the comment completely, as `timeout-minutes` may be clear enough on its own?